runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/demo.json

```diff
@@ -0,0 +1,5 @@
+{
+    "Demo.Process.Ran.Pt1": "Proces uruchomiony:",
+    "Demo.Process.Ran.Pt2": "razy",
+    "Demo.Session.Timer": "Zegar Sesji:"
```

Comment:
WDYT about "Pozostało:" lub "Do końca sesji:"
odpowiednio remaing/session ends in